### ISSUE TITLE
Enhancement: Adding `--no-schema` option to `litestar routes` command

### ISSUE THREAD

**Issue by solzard** [litestar-org/litestar#1614]
### Summary

Add `--no-schema` option to `litestar routes` to hide schema routes.

### Basic Example

`litestar routes` currently display all routes information including schemata.

```shell
> litestar routes

Using Litestar app from app:app
/ (HTTP)
├── heartbeat async GET
└── options_handler sync OPTIONS
/detect (HTTP)
├── detect async POST
└── options_handler sync OPTIONS
/detect-without-roi (HTTP)
├── detect_without_roi async POST
└── options_handler sync OPTIONS
/schema (HTTP)
├── root sync GET
└── options_handler sync OPTIONS
/schema/elements (HTTP)
├── stoplight_elements sync GET
└── options_handler sync OPTIONS
/schema/openapi.json (HTTP)
├── retrieve_schema_json sync GET
└── options_handler sync OPTIONS
/schema/openapi.yaml (HTTP)
├── retrieve_schema_yaml sync GET
└── options_handler sync OPTIONS
/schema/redoc (HTTP)
├── redoc sync GET
└── options_handler sync OPTIONS
/schema/swagger (HTTP)
├── swagger_ui sync GET
└── options_handler sync OPTIONS
```

Sometimes, I just want to glance at routes only **I** built, not including schema routes Litestar automatically generated.

Adding an option such as `--no-schema` to `litestar routes` would be appreciated.

```shell
> litestar routes --no-schema

Using Litestar app from app:app
/ (HTTP)
├── heartbeat async GET
└── options_handler sync OPTIONS
/detect (HTTP)
├── detect async POST
└── options_handler sync OPTIONS
/detect-without-roi (HTTP)
├── detect_without_roi async POST
└── options_handler sync OPTIONS
```

I am happy to implement this within a day or two if the idea gets no objection 😀

### Drawbacks and Impact

Since it is an option, it will not bring any backward incompatible behaviors.

<a href="https://polar.sh/litestar-org/litestar/issues/1614">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/1614/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/1614/pledge.svg">
</picture>
</a>

**Comment by JacobCoffee** [comment]
Open for anyone

**Comment by andrewdoh** [comment]
> Open for anyone

could i take it? I have something here.  https://github.com/litestar-org/litestar/pull/2886

**Comment by peterschutt** [comment]
Closed in #2886
